west.yml

```diff
@@ -0,0 +1,25 @@
+# The west manifest file for downstream Zephyr module ZXA.
+#
+# The per-installation west configuration file, .west/config, sets the
+# path to the project containing this file in the [manifest] section's
+# "path" variable.
+#
+# This imports the latest upstream Zephyr manifest file.
+
+manifest:
+  version: 0.7
+
+  defaults:
+    remote: zephyrproject-rtos
+
+  remotes:
+    - name: zephyrproject-rtos
+      url-base: https://github.com/zephyrproject-rtos
+
+  #
+  # Please add items below based on alphabetical order
+  projects:
+    - name: zephyr
+      remote: zephyrproject-rtos
```

Comment:
this is not necessary, since it's the default remote

Reply:
This was provided as an example of a field that must be set when using a fork of zephyr rather than zephyr itself (as provided by some vendors).  Create the second example repository and we'll not have it in the simple one.